application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -25,33 +28,35 @@
  * Commands need to be added to the commandList
  */
 public class CommandHandler extends ListenerAdapter {
-    private final static Logger logger = LoggerFactory.getLogger(CommandHandler.class);
+    private static final Logger logger = LoggerFactory.getLogger(CommandHandler.class);
 
     private final List<Command> commandList = new ArrayList<>();
     private final Map<String, Command> commandMap;
 
-    public CommandHandler() {
-        commandList.addAll(List.of(new ReloadCommand(this)));
+    public CommandHandler(Database database) {
+        commandList.addAll(
+                List.of(new ReloadCommand(this), new PingCommand(), new DatabaseCommand(database)
+                // add your command here
+                ));
 
         commandMap = commandList.stream()
             .collect(Collectors.toMap(Command::getCommandName, Function.identity()));
     }
 
     @Override
     public void onReady(@NotNull ReadyEvent event) {
-        event.getJDA().getGuildCache().forEach(guild -> {
-            guild.retrieveCommands().queue(commands -> {
-                boolean hasReloadCommand =
-                        commands.stream().anyMatch(command -> command.getName().equals("reload"));
+        event.getJDA().getGuildCache().forEach(guild -> guild.retrieveCommands().queue(commands -> {
```

Comment:
what are the changes here? looks like u only applied formatting. is the formatting correct according to spotless? was it wrong before?

Reply:
I believe it was either done by Spotless or by IJ's auto-formatting.

The only change I did myself was removing the unnecessary brackets